ci-operator/step-registry/openshift/e2e/aws/builds/techpreview/openshift-e2e-aws-builds-techpreview-workflow.yaml

```
@@ -0,0 +1,14 @@
+workflow:
+  as: openshift-e2e-aws-builds-techpreview
+  steps:
+    pre:
+    - chain: ipi-aws-pre-techpreview
+    test:
```

We also have a step that adds the github secrets needed for private git tests:
```suggestion
    - chain: ipi-aws-pre-techpreview
    - ref: build-github-secrets
    test:
```